rego-templates/embedded.go

```diff
@@ -0,0 +1,101 @@
+package rego_templates
+
+import (
+	"bufio"
+	"embed"
+	"io/fs"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/aquasecurity/postee/v2/data"
+	"github.com/aquasecurity/postee/v2/log"
+)
+
+const (
+	COMMON_DIR               = "common"
```

Comment:
NIT: I also have java background and can understand uppercase in constant names, but go has different convention